docs/keyboardShortcuts.md

```diff
@@ -62,8 +62,9 @@ These shortcuts can be used when the focus is on an issue card in a panel.
 
 ## Control the panel view window ...
 
-| **Action on the browser view**| **Shortcut**|**Explanation**|
+| **Action on the panel view**| **Shortcut**|**Explanation**|
 | ------------- |:-------------:| -----------|
+| Bring HubTurbo into focus|<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>H</kbd>| Global Hotkey to make HubTurbo topmost window|
```

Comment:
Can we use this shortcut for both dismissing and restoring? i.e. if not in focus (or minimized), it restores both windows and give focus to pView. If pView is in focus, it minimized both windows.